Automattic/wp-calypso
# Reader: add more information to signup landing page

#### Changes proposed in this Pull Request

Highlight Reader features in the signup landing page, based on new design by @jancavan.

#### Testing instructions

Visit http://calypso.localhost:3000/start/reader. Check the landing page appears correctly on a variety of browsers and at different screen widths.



Desktop:

![reader-landing-page-desktop](https://user-images.githubusercontent.com/17325/47272972-9f35ce80-d5e9-11e8-9c49-c6f627359a52.png)

Mobile:
![reader-landing-page-mobile1](https://user-images.githubusercontent.com/17325/47272973-ab219080-d5e9-11e8-8412-e8bfe29f0ab1.png)
